go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/aws/protocol/eventstream v1.7.2
 	github.com/aws/aws-sdk-go-v2/config v1.31.13
 	github.com/aws/aws-sdk-go-v2/service/sts v1.38.7
+	github.com/barkimedes/go-deepcopy v0.0.0-20220514131651-17c30cfc62df
```

Comment:
drive-by: I would not accept a random direct dependency like this that is not well maintained or very popular either